.github/workflows/build.yml

```diff
@@ -443,8 +491,12 @@ jobs:
           pytest -v -s tests/solvers/python
           pytest -v -s tests/scheduling
 
-  build-docs:
-    needs: [test-linux, test-macos, test-windows]
+  build-doc:
+    needs:
+      - test-ubuntu
+      - test-macos
+      - test-windows
```

Comment:
Why do you suddenly decide that 'needs' should be written on multiple lines?  Previous version looks more readable to me